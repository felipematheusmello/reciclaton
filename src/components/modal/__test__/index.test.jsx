import React from "react";
import Modal from "../index";
import renderer from "react-test-renderer";

it("Modal Test", () => {
  const tree = renderer.create(<Modal>Ai ai esse CRL viu...</Modal>).toJSON();
  expect(tree).toMatchSnapshot();
});
