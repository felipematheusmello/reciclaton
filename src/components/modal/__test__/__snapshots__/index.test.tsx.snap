// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Test 1`] = `
<div>
  <div
    className="sc-bdnylx fifHaM"
  >
    <button
      className="submit"
    >
      <a
        href="#modal"
      >
        Ai ai esse CRL viu...
      </a>
    </button>
  </div>
  <div
    className="sc-gtssRu cJuzsI"
    id="modal"
  />
  <form
    className="box"
    name="form"
  >
    <label>
      Nome:
    </label>
    <input
      id="name"
      name="name"
      type="text"
    />
    <label>
      Usuário:
    </label>
    <input
      id="user"
      name="user"
      type="text"
    />
    <label>
      Email:
    </label>
    <input
      id="email"
      name="email"
      type="email"
    />
    <label>
      Senha:
    </label>
    <input
      id="pass"
      name="pass"
      type="password"
    />
    <label>
      Confirmação de senha:
    </label>
    <input
      id="confirm"
      name="confirm"
      type="password"
    />
    <input
      id="submit"
      type="submit"
      value="Enviar"
    />
    <a
      href=""
      id="close"
    >
      x
    </a>
  </form>
</div>
`;
